py/desispec/scripts/purge_tilenight.py

```diff
@@ -48,12 +51,20 @@ def remove_directory(dirname, dry_run=True):
         if dry_run:
             log.info(f"Dry_run set, so not performing any action.")
         else:
+            if not no_attic:
+                if dirname.startswith('/'):  # absolute path
+                    attic_dir = dirname.replace(specprod_root(),
+                        os.path.join(specprod_root(), 'attic'))
+                else:  # relative path
+                    attic_dir = os.path.join('attic', dirname)
+                log.info(f"Copying {dirname} to attic")
+                shutil.copytree(dirname, attic_dir, dirs_exist_ok=True)
```

Comment:
We use symlinks at a few points in the pipeline, particularly in the /exposures/ file. Looking at the `copytree` documentation it replaces symlinks with copies of the data by default. However, I don't think using `symlinks=True` will work because it will likely assign those links to the old filepaths that will then be removed with `shutil.rmtree`. The relative links might continue to work, but not for the `fiberflatexp-*` files that link to calibnight (which won't be purged in the case of `purge_tilenight`).

Because we're talking about a small amount of data, I think the code is fine as-is. I just wanted to point this out in case it is relevant in the future.

Reply:
Anthony and I talked in person, and we agree that to save disk space we will set `symlinks=True` and not make extra copies of the symlinked files. That would probably make the attic symlinks invalid, but we think that's fine.

Reply:
I have modified the code such that the attic symlinks will have the same absolute path as the original ones. So we should be able to run code in the attic without any issues.